packages/thirdweb/src/react/native/ui/connect/InAppWalletUI.tsx

```diff
@@ -62,9 +68,8 @@ export function InAppWalletUI(props: InAppWalletFormUIProps) {
   const { wallet, theme } = props;
   const config = wallet.getConfig();
   const authOptions = config?.auth?.options || defaultAuthOptions;
-  const socialLogins = authOptions.filter(
-    (x) =>
-      x === "google" || x === "apple" || x === "facebook" || x === "discord",
+  const socialLogins = authOptions.filter((x) =>
+    socialAuthOptions.includes(x as SocialAuthOption),
```

Comment:
weird is this `as` necessary? why? shouldn't typescript 5.5 handle this?

Reply:
Yeah been seeing this a lot, haven't dug into why